hadoop-ozone/common/src/main/resources/ozone-version-info.properties

```diff
@@ -19,9 +19,6 @@
 version=${declared.ozone.version}
 release=${ozone.release}
 revision=${version-info.scm.commit}
-branch=${version-info.scm.branch}
-user=${user.name}
-date=${version-info.build.time}
 url=${version-info.scm.uri}
```

Comment:
sparing commit id and url means the build should happen from official apache repo itself  i.e we cannot compare across forks or with same src code but with different commit chain?